unittests/backends/qpp_observe/QppObserveTester.cpp

```diff
@@ -27,7 +27,7 @@ CUDAQ_TEST(QPPBackendTester, checkBackendObserve) {
                      .21829 * z(0) - 6.125 * z(1);
 
   auto expVal = qpp.observe(h);
-  EXPECT_NEAR(expVal.expectationValue.value(), -1.74, 1e-2);
+  EXPECT_NEAR(expVal.expectation(), -1.74, 1e-2);
```

Comment:
Is this an interface change that is visible to the user?

Reply:
This is not visible to regular users unless they retrieve the `CircuitSimulator` instance in user code, e.g., via `cudaq::getUniquePluginInstance` and interact with the simulator directly.